models/victoriametrics.go

```diff
@@ -0,0 +1,124 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package models
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+
+	config "github.com/percona/promconfig"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+
+	// enables cache for vmdb responses.
+	vmCacheEnableEnv = "VM_CACHE_ENABLE"
+	// enables victoriametrics services at supervisor config.
+	vmTestEnableEnv = "VM_TEST_ENABLE"
```

Comment:
can we rename this env variable to `PERCONA_TEST_ENABLE_VM`?

Reply:
Yep, sure.